### DISCUSSION TITLE
Possible to add the date associated with a Quarto version to the download page?

### DISCUSSION THREAD

**Discussion by coatless** [quarto-dev/quarto-cli#7549] [Feature Requests]
### Description

The download page for both the **release** and **pre-release** lack a date associated with them. Would it be possible to add a date next to the download button? Similar to how the RStudio-Desktop build shows a date?

<img width="615" alt="Date associated with RStudio build" src="https://github.com/quarto-dev/quarto-cli/assets/833642/c1edf0ca-6d55-4f2b-8ead-58921b853317">

https://posit.co/download/rstudio-desktop/

One reason this is coming up is the pre-release seems to be stuck at a version released 3 days back.

**Comment by mcanouil** [comment]
What do mean by pre-release 3 days back?
There were no prerelease build since the 7th of November, so the last is the one showed.
See <https://github.com/quarto-dev/quarto-cli/releases>.
Pre-release build is not daily build.

Note that if you really want to be up to date to the last commit on main, then you can download and compile from source (see repository readme).

**Comment by coatless** [comment]
From the date of the post (11/10), the latest pre-release was triggered on 11/7. So, about 3 days. 😄 

On the other note, I think I was under the impression the pre-release builds were either nightly or being triggered after PR merges given the high number of releases (~1.5k). So, it seemed a bit odd that the pre-release page didn't contain an update after 3 days... 

https://quarto.org/docs/download/prerelease

Thus, the request over for including a date on both the pages. 

Since this is a bit public, I suppose the question here is whether a PR would be welcomed to add the date information stored in JSON?

- [`download/_prerelease.json`](https://github.com/quarto-dev/quarto-web/blob/bab7b7e440ac4f61051988374f7b3d07264d2399/docs/download/_prerelease.json#L5)
- [`download/index.qmd`](https://github.com/quarto-dev/quarto-web/blob/bab7b7e440ac4f61051988374f7b3d07264d2399/docs/download/index.qmd)

**Reply by mcanouil** [reply]
I can't check right now, but I am pretty sure that's not the file to modify, i.e., I believe the releases are retrieved from the CI into the json and likely the "include" Quarti document.

Adding the date won't change the fact that it is up to date and that there is no daily/nightly (release) build.

Note that I personally don't mind having or not having the date (as I don't see the usefulness of it).
This being said, nothing forbids you to make a PR if you think that's appropriate. The team will review it and merge or not.

**Reply by coatless** [reply]
@mcanouil I hear you on the nightly part. 

The reason for the date is two-fold:

1. Being able to understand when a new pre-release or patched version of the release was released vs. looking at the GH release page.
2. Simplifies the discussion with folks who want to use a feature that if you haven't updated your pre-release version of Quarto since MM/DD/YYYY, then you need to update. (I can't bump an extension's minimum requirement into pre-release territory.) 

On the note of the PR, I was hoping to get a 👍 or 👎 on the feature before investing in learning more about how the download page was updated.

**Reply by mcanouil** [reply]
I would say go for it as I don't see any good reason to not add the date.

**Reply by coatless** [reply]
Okidokie. Submitted the change in: https://github.com/quarto-dev/quarto-web/pull/915
